docs/Migrating_LLB_to_a_PPL_based_Custom_Device.md

```diff
@@ -0,0 +1,74 @@
+## Migrating a Custom Device from LLB to PPL
+
+First introduced with LabVIEW 2010, a packed project library is a compiled .lvlib that contains all of the .lvlib’s VIs and allows the user to call the public VIs in a manner that is identical to the use of the original .lvlib. Since 
+LabVIEW 2017 you can compile a packed project library to be also used with newer versions of LabVIEW. This option is enabled by default, but if you want to change that setting, choose "Advanced" in the sidebar and disable the option "Allow future versions of LabVIEW" to load this packed library. 
+Using PPLs for packaging VeriStand Custom Devices does not eliminate the usage of LLBs for the same purpose. A developer does not have to chose between using one or another. The two options can coexist as part of the same project. A developer can keep the existing LLB build specifications and simply add to the project the additional PPL ones. Of course, it might be necessary to modify the CD source code in order to be able to support both options - this will be discussed below. The XML configuration file would have to be modified as well to reflect the new loading paths for the CD elements. 
+
+### Advantages of using PPL based Custom Devices
+
+Packed Project Libraries (PPL) provide name spacing for all of its contained items and preserve the file hierarchy of the source project library. These features result in several potential benefits for a Custom Device packed in a PPL:
+* Even if two given custom devices would reference the same file (VI) dependency, by packaging them into PPLs, each custom device would receive its own copy of the required dependency. This would be compiled and included in its respective PPL via name spacing for the contained items, the PPLs would also make sure that load-time conflicts are avoided.
```

Comment:
```suggestion
* If two given custom devices would reference the same VI dependency, by packaging them into PPLs, each custom device would receive its own copy of the required dependency. This would be compiled and included in its respective PPL via namespacing for the contained items. The PPLs would also make sure that load-time conflicts are avoided.
```